shell/platform/darwin/ios/framework/Source/accessibility_value_provider.h

```diff
@@ -0,0 +1,18 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
```

Comment:
I know you added this for testability, but it seems like overkill to add a new header and implementation file for a single wrapper around a function.  `FlutterPlatformViews_Internal` might be a better spot.

Reply:
I tried adding it to `FlutterPlatformViews_Internal`, but in the end, failed to do so. I struggled with some ARC-related errors coming in from the underlying imports having an effect on existing test cases. 

Instead of going for `FlutterPlatformViews`, I was able to add it to the `FlutterViewController_Internal` instead. Hope that will also work for you. Please let me know!

Reply:
`FlutterViewController_Internal.mm` doesn't make sense, the implementation is already "internal", the `_Internal.h` suffix is used to indicate it's not a "public" header and is not exported in the module.

What ARC errors were you seeing?

Reply:
I pasted `AccessibilityValueProvider` at the bottom of `FlutterViewController.m` and it compiled fine.  You can push up your change if you want me to take a look though.

Reply:
Ah that's also an option ofcourse, I didn't consider that one yet. Thanks! Will take a look.